<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCTCPLinkConfiguration</class>
 <widget class="QWidget" name="QGCTCPLinkConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::FieldsStayAtSizeHint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="portLabel">
     <property name="text">
      <string>TCP Port</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="portSpinBox">
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Host Address</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="hostAddressLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="asServerCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>As Server?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
